tensorflow/lite/arena_planner.cc

```diff
@@ -284,7 +284,10 @@ TfLiteStatus ArenaPlanner::CalculateTensorDeallocation(int tensor_index) {
 
 TfLiteStatus ArenaPlanner::CalculateAllocationOfInternalTensors(
     int node_index) {
-  if (node_index < graph_info_->num_nodes()) {
+  if (node_index < 0) {
```

Comment:
It's probably worth a comment as to when node_index would be negative.

Reply:
@jdduke thanks for the review, i have updated the comments to describe the problem, kindly check and approve.

Regards
Amit